### PR TITLE
Markdown guide

### PR DESCRIPTION
**Pull request by peteryates**
Add a basic Markdown guide and add links to supporting docs in the README

### CODE REVIEW

File: supporting-documents/learning-markdown.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
+# Learning Markdown 
+
+Markdown supports a very limited subset of is _incredibly simple_.
```
Comment (rjlynch) [inline supporting-documents/learning-markdown.md]: Is this typo'd?

**Review by rjlynch — CHANGES_REQUESTED** [review]
Looks good, one minor typo

File: supporting-documents/learning-markdown.md (hunk 1 above)
Comment (peteryates) [reply supporting-documents/learning-markdown.md]: Yep! 😳

**Review by rjlynch — APPROVED** [review]
👍 Very nice

**Pull request merged**